lib/src/core/authentication-core.ts

```diff
@@ -93,8 +96,8 @@ export class AuthenticationCore<T> {
         }
 
         if (configData.enablePKCE) {
-            const codeVerifier = CryptoUtils.getCodeVerifier(configData.runtimeEnvironment!);
-            const codeChallenge = CryptoUtils.getCodeChallenge(codeVerifier);
+            const codeVerifier = this._cryptoUtils.getCodeVerifier(configData.runtimeEnvironment!);
```

Comment:
Can we do a null assertion here?

Reply:
And we don't need the argument anymore, do we?

Reply:
Fixed